msbuild/Xamarin.iOS.Tasks.Core/Tasks/DetectSdkLocationsTaskBase.cs

```diff
@@ -287,19 +288,26 @@ void EnsureiOSSdkPath ()
 			SdkPlatform = SdkIsSimulator ? "iPhoneSimulator" : "iPhoneOS";
 		}
 
-		void EnsureAppleSdkRoot ()
+		bool EnsureAppleSdkRoot ()
 		{
 			if (!CurrentSdk.IsInstalled) {
-				Log.LogError ("  Could not find a usable Xcode app bundle in {0}", CurrentSdk.DeveloperRoot);
-			} else {
-				Log.LogMessage (MessageImportance.Low, "  DeveloperRoot: {0}", CurrentSdk.DeveloperRoot);
-				Log.LogMessage (MessageImportance.Low, "  DevicePlatform: {0}", CurrentSdk.DevicePlatform);
-				Log.LogMessage (MessageImportance.Low, "  GetPlatformPath: {0}", CurrentSdk.GetPlatformPath (false));
-
-				SdkDevPath = CurrentSdk.DeveloperRoot;
-				if (string.IsNullOrEmpty (SdkDevPath))
-					Log.LogError ("  Could not find valid a usable Xcode developer path");
+				var ideSdkPath = "(Project > SDK Locations > Apple > Apple SDK)";
+#if WINDOWS
+				ideSdkPath = "(Tools > Options > Xamarin > iOS Settings > Apple SDK)";
+#endif
```

Comment:
minor: you're (re)setting the value `ideSdkPath` on windows. You can do something like:

```
#if WINDOWS
var ideSdkPath = "...";
#else
var ideSdkPath = "...";
#endif
```